src/output.rs

```diff
@@ -316,10 +316,35 @@ pub(crate) fn write_buffer(buffer: &[u8], f: &mut fmt::Formatter<'_>) -> fmt::Re
     if let Ok(buffer) = str::from_utf8(buffer) {
         write!(f, "{}", buffer)
     } else {
-        write!(f, "{:?}", buffer)
+        write_bytes(buffer, f)
     }
 }
 
+pub(crate) fn write_bytes(buffer: &[u8], f: &mut fmt::Formatter<'_>) -> fmt::Result {
+    write!(f, "b\"")?;
+    for &b in buffer {
+        let repl = match b {
+            b'\n' => "\\n",
```

Comment:
If it is utf8, we don't print it using debug.  Should we do the same here and once we have `from_utf8_lossy`, just print that as-is?

If not, should we just do a `"{:?"}"` on the lossy data?  I'd prefer keeping the code simple / reusing the stdlib.

Reply:
Now use `bstr`